src/components/SignInForm.tsx

```diff
@@ -20,7 +20,9 @@ const Card = styled(MuiCard)(({ theme }) => ({
   maxWidth: 'min(450px, 90dvw)',
   padding: theme.spacing(4),
   gap: theme.spacing(2),
-  marginTop: '25%',
+  [theme.breakpoints.up('sm')]: {
+    maxWidth: '450px',
+  },
```

Comment:
I'm guessing something went wrong here when you merged main into your branch? I have set a maxWidth a couple rows up which should work for all breakponts
```suggestion
marginTop: '25%',
```

Reply:
I think that's just some leftover code from a template I used and forgot to remove it from my branch